.github/workflows/rerun-flaky-workflows.yml

```diff
@@ -2,7 +2,7 @@ name: Rerun Flaky Workflows
 
 on:
   workflow_run:
-    workflows: ["K8s-CI", "CNVM-CI"]
+    workflows: ["K8s-CI", "CNVM-CI", "Tests"]
```

Comment:
shouldn't this be just `Tests` now?

Reply:
For the `main` branch, you are correct. However, since we haven't backported the refactored CI, these workflows remain relevant for other branches.